packages/react-ui-core/src/Form/SpecificFields/Message.js

```diff
@@ -1,12 +1,9 @@
 import React, { PureComponent } from 'react'
 import PropTypes from 'prop-types'
 import themed from 'react-themed'
-import { Field } from '../../Form'
-
-@themed(['Label_Textarea'], {
-  pure: true,
-})
+import Field from '../Field'
 
+@themed(['Message', 'Label_Textarea'], { pure: true })
```

Comment:
Weird. This should have never been themed in the first place.  `Label_Textarea` isn't in this component so it's not needed.

Reply:
I'll remove it. I was just making sure i didn't break something or make it backwards incompatible.